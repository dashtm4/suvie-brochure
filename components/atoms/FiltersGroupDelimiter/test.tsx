import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { FiltersGroupDelimiter } from '.'

describe('<FiltersGroupDelimiter />', () => {
  it('should renders correct <FiltersGroupDelimiter />', () => {
    const wrapper = shallow(<FiltersGroupDelimiter />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
