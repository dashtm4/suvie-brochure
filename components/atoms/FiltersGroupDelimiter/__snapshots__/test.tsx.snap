// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FiltersGroupDelimiter /> should renders correct <FiltersGroupDelimiter /> 1`] = `
<div
  className="hidden md:block md:border md:border-left md:border-secondary"
/>
`;
